ZW096 update (re-add needed)

caseda opened the pull request:
app.json:
proper value setting "3"
add default setting "90"
change default setting "91" size => 2
remove default setting "101" default anyway
remove default setting "102" default anyway
add default setting "111"
add default setting "112"
add meter_power
english vs american color => colour
change min/max to inside attribute
remove setting "101" can stop data to homey
remove setting "102" can stop data to homey
change hint setting "111"
change min/max setting "111" to inside attribute
change hint setting "112"
change min/max setting "112" to inside attribute

driver.js:
coding consistency
add meter_power capability
remove setting parser
remove setting "101" can stop data to homey

RobinBol commented:
Why would you remove association group 2? (re-added it)
There is no property 'Scale' in Properties1, so meter_power and measure_power are broken.
And scale bits 2 = W, 1 = kWh, not the other way around; now they report to the wrong capability.

You say: 'remove setting "102" can stop data to homey' but it is not removed?
You also say: 'remove setting "101" can stop data to homey' this one is actually removed, but I re-added it, don't see why you would want to remove it.

caseda commented:
there has been some changes when working on the driver itself:

i removed the id from group 2 because i combined the measure_power and meter_power into group 1, see commit:
https://github.com/athombv/com.aeotec/pull/18/commits/6f78424e73b422b58b29a96b40e4ff4c9a7d248d

My bad about the Scale in Properties1, i've noticed there is a big difference in version 1&2 and versions 3&4, but only afterwards (too late)

this is also why i initially removed 102, and after re-added it again because both are coming in on group 1 now so don't give null back.
i removed 101 cause it will give null back to the mobile card for the meter's that they don't select, people will complain if it stays on null :)
But if you don't find that a reason good enough 👍

RobinBol commented:
Ok thanks!

Association group 2: "Forward the Basic Set, Switch Binary Set to associated nodes in Group 2 when the Smart Switch receives the Basic Set, Switch Binary Set commands from main controller." So it could be useful I guess, will leave it on by default for now.
